src/components/common/Header.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./index.css";
+import { Image, Container } from "react-bootstrap";
+import profilePhotoButton from "../../images/profilePhotoButton.png";
+
+const Header = (props) => {
+  const { headerBG, imagePath, headerPage, isProfile, extraComponents } = props;
+  return (
+    <div
+      className="dynamicHeader"
+      style={{ backgroundImage: `url(${headerBG})` }}
+    >
+      <Container className={headerPage}>
+        {isProfile ? (
+          <div className={headerPage + "ImgContainer"}>
```

Comment:
for clarity this code would need some comments about what headerPage can contain, or a change in variable names. for example right now i have no idea what the `headerPage` variable contains. is it a string of sorts that represents something?